doc/kci_testsuite.md

```diff
@@ -0,0 +1,289 @@
+### Verify your requirement:
```

Comment:
Please add a paragraph explaining what this page is all about, i.e. how to add a test suite to KernelCI.